CHANGELOG.md

```diff
@@ -1,5 +1,71 @@
 # [Versions](https://mui.com/versions/)
 
+## v5.15.8<!-- generated comparing v5.15.7..master -->
+
+_Feb 5, 2024_
+
+A big thanks to the 15 contributors who made this release possible. Here are some highlights ✨:
+
+- ✨ Added support for Arrow Down/Up + Shift and Page Up/Down keys for faster stepping in the Slider component (#40676) @mnajdova
+- many 🐛 bug fixes and 📚 documentation improvements
+
+### `@mui/material@5.15.8`
+
+- &#8203;<!-- 36 -->[Avatar] Simplify valid children assertion (#40834) @oliviertassinari
+
+### `@mui/codemod@5.15.8`
+
+- &#8203;<!-- 37 -->[Accordion] Update props actual.js test case (#40879) @DiegoAndai
+- &#8203;<!-- 12 -->Fix `findComponentJSX` util (#40855) @sai6855
+
+### `@mui/system@5.15.8`
+
+- &#8203;<!-- 10 -->Add blend color manipulator (#40258) @romgrk
+- &#8203;<!-- 02 -->[zero] Always replace the `createUseThemeProps` call (#40885) @siriwatknp
+- &#8203;<!-- 01 -->[zero] Add README with installation and basic usage (#40761) @brijeshb42
```

Comment:
Shouldn't these two be under zero-runtime section?

Reply:
Currently we keep all zero-related changes under the system, once we release an alpha version, we can add them under dedicated header.

Reply:
I followed what was done in previous versions, e.g. https://github.com/mui/material-ui/blob/296df99c40cd32a4859d48fddeec0c2b1cfce449/CHANGELOG.md?plain=1#L92